serverless/src/exportSearchCheck/__tests__/handler.test.js

```diff
@@ -0,0 +1,208 @@
+import knex from 'knex'
+import mockKnex from 'mock-knex'
+import MockDate from 'mockdate'
+
+import * as getDbConnection from '../../util/database/getDbConnection'
+import * as deployedEnvironment from '../../../../sharedUtils/deployedEnvironment'
+import * as getJwtToken from '../../util/getJwtToken'
+import * as getVerifiedJwtToken from '../../util/getVerifiedJwtToken'
+
+import exportSearchCheck from '../handler'
+
+const OLD_ENV = process.env
+
+const MOCK_EXPORT_REQUEST_KEY = 'MOCK_EXPORT_REQUEST_KEY'
+const MOCK_USER_ID = 1234
+const MOCK_SIGNED_URL = 'MOCK_SIGNED_URL'
+
+let dbTracker
+
+jest.mock('aws-sdk', () => {
+  const instance = {
+    headObject: jest.fn(() => ({
+      promise: jest.fn().mockResolvedValue(true)
+    })),
+    // jest.fn().mockResolvedValue(MOCK_SIGNED_URL) returns Promise { undefined }
```

Comment:
commented out code

Reply:
Sorry, I wrote that in a confusing way. I've tried to clarify my comment with this https://github.com/nasa/earthdata-search/pull/1630/commits/37efdbfba47ce571ff49b88a5e79a6bb8909871e.  Let me know if that works.

I'm basically trying to explain why I don't call mockResolvedValue, which would be the "best practice" way to do this.  It appears that there's some sort of bug in jest or I'm just not understanding how to use mockResolvedValue.